"""
Bunch of functions
"""

def add(number_a:int, number_b:int):
    """
    This function adds 2 numbers
    """
    print( number_a  + number_b )
    
    
"""
There is no error in the code. But we can add a print inplace of return the show the output.

"""
